Tests/AutoTests/FunctionalTest/functional-weeklyfiledownload.spec.ts

```diff
@@ -61,4 +67,15 @@ test.describe("Goto maritime-safety-information Notice To Mariner Page to Check
   await noticeFileDownload.verifyleisureFoliosFileNameDownload();
   
 })
+
+test('Should Goto Notices to Mariner Page for Daily download File with Distributor Login', async ({ page, context }) => {
```

Comment:
It's confusing that these daily download tests are in a file called "weeklyfiledownload".  Can they be moved?

Reply:
Resolved